packages/@aws-cdk/cx-api/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@aws-cdk/cx-api",
   "version": "0.0.0",
-  "private": true,
```

Comment:
We need a deep discussion about cx-api and cloudassembly-schema and how exactly they are vended and consumed by the monocdk (copy @iliapolo)